src/layouts/Base.astro

```diff
@@ -3,10 +3,11 @@ import { Footer, Header } from '~components';
 
 export interface Props {
 	subtitle?: string;
+	skipTargetId?: string;
 	title: string;
 }
 
-const { title } = Astro.props as Props;
+const { title, skipTargetId } = Astro.props as Props;
```

Comment:
🔍 Adding `skipTargetId` to the `Base` layout allows us to provide an IDREF to the heading we want to target on each page